docs/3.main-concepts/9_1.webchat.mdx

```diff
@@ -0,0 +1,90 @@
+---
+name: 10. Webchat
+route: /main-concepts/webchat
+menu: 3. Main Concepts
+---
+
+# **Webchat**
+
+## What is Webchat?
+
+Bots developed with Botonic can be easily deployed to different messaging channels. 
+This is due to the fact that Botonic components are specifically treated in order to be executed in a server environment such as Node.Js. 
+As they are based on React, Botonic components are the perfect fit in order to offer you applications such as Webchat. Webchat is a system that allows you to not being restricted to use one of the different messaging providers in order to be able to run your bots and it can easily be embedded in your website.
+It has all the necessary features that you will need to develop a conversational application, such as interactive elements, handovers to human agents, etc.
+
+<img
+	src="https://botonic-doc-static.netlify.com/images/webchat_usage.png"
+	style={{ display: 'block', margin: 'auto', maxWidth: '50%' }}
+/>
+
+_Webchat's look. Embedded Webchats can be triggered by clicking the chat bubble._
+
+
+
+## Webchat SDK API
+
+Once you have your Webchat integrated ([see Webchat channel integration](/main-concepts/channels)), you will be provided with a snippet of code to be embedded in your page.
+This snippet of code will be loading as a script the entire code of your bot and thus, the following API methods will be available for you in order to invoke explicitly the Webchat component (you can test it in your current browser's developers console).
```

Comment:
>This snippet of code will be loading as a script the entire code of your bot

 This snippet will load your bot's code and will allow managing it though the following JS functions (you can test it in your current browser's developers console)